packages/api/src/hooks/useSendFiles.ts

```diff
@@ -12,14 +12,14 @@ type PostActivityFile = {
   url: string;
 };
 
-export default function useSendFiles(): (files: PostActivityFile[]) => void {
+export default function useSendFiles(): (files: PostActivityFile[], text?: string) => void {
```

Comment:
When `text` argument is introduced to `useSendFiles`, it almost act the same as `useSendMessage`.

Instead of adding `text` argument to `useSendFiles`, should add `attachments` to `useSendMessage` instead. Then, `useSendFiles` can be deprecated.

Reply:
Added deprecation notes.